tools/dockerfile/interoptest/grpc_interop_php/build_interop.sh

```diff
@@ -36,6 +36,12 @@ make -j4 install_c plugins
 # Install protobuf (need access to protoc)
 (cd third_party/protobuf && make -j4 install)
 
-(cd src/php && php -d extension=ext/grpc/modules/grpc.so /usr/local/bin/composer install)
+cd src/php
 
-(cd src/php && ./bin/generate_proto_php.sh)
+set +e
+php -d extension=ext/grpc/modules/grpc.so /usr/local/bin/composer install
+while [ $? -ne 0 ]; do
+  php -d extension=ext/grpc/modules/grpc.so /usr/local/bin/composer install
```

Comment:
I'm a little worried about this while loop causing an infinite retry

Can we limit the retries with something like:

```
set -ex
DONE=0
for ((i=0;i<10;i++)); do
  php -d extension=ext/grpc/modules/grpc.so /usr/local/bin/composer install && DONE=1
  [[ "$DONE" == 1 ]] && break
done
[[ "$DONE" != 1 ]] && echo "Failed to install" && exit 1
```

or 

```
php -d extension=ext/grpc/modules/grpc.so /usr/local/bin/composer install || php -d extension=ext/grpc/modules/grpc.so /usr/local/bin/composer install || php -d extension=ext/grpc/modules/grpc.so /usr/local/bin/composer install
```

?